src/makefile.osx

```diff
@@ -130,7 +136,7 @@ DEFS += $(addprefix -I,$(CURDIR)/leveldb/include)
 DEFS += $(addprefix -I,$(CURDIR)/leveldb/helpers)
 OBJS += obj/txdb-leveldb.o
 leveldb/libleveldb.a:
-	@echo "Building LevelDB ..."; cd leveldb; make; cd ..
+	@echo "Building LevelDB ..."; cd leveldb; make libleveldb.a libmemenv.a; cd ..;
```

Comment:
When we compile the source code to generate the deamon `Electrad` we also need `libmemenv.a`. Would it be possible to also build LevelDB on `make -f makefile.osx Electrad`?